CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,10 @@
 == 3030-12-31 Getting Started
 <p>If you're new to CONSTELLATION, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2021-02-02 Blaze Colors - Presets and Recents
+<p>The blaze menus have been updated to show menus for recent and preset blaze colors.</p>
+<p>The preferences of Constellation now keep track of what colors you recently used as well as the chance to preset blaze colors via color picker.</p>
```

Comment:
Are you able to add some instructions for the getting to the preset colour picker in preferences? Wasn't clear from this exactly where I can go to add the preset blaze colours

Reply:
I've committed an update: but in short it is now :
`The preferences found via Setup -> Options -> CONSTELLATION -> Graph now keeps track of what colors you recently used as well as the chance to preset blaze colors via color picker.`